docker/yb-voyager-docker

```diff
@@ -79,22 +117,27 @@ do
 			done
 		else 
 			# If the data-dir is not an S3 bucket
-			# If the OS is macOS
 			data_dir=$(realpath "$data_dir")
-			if [[ $OSTYPE == 'darwin'* ]] 
-			then
-				# Delete the old volume and create a new one it its place attached to the user's data-dir
-				docker volume rm voyager-data-dir
-				docker volume create --driver local --opt type=none --opt device="${data_dir}" --opt o=bind voyager-data-dir 
-		
-				data_vol="-v voyager-data-dir:/data-dir"
-			else
-				data_vol="-v ${data_dir}:/data-dir"
-			fi
-			argv[$j]='/data-dir'
+			map_host_path_inside_container "data-dir" "${data_dir}"
 		fi
 	fi	
 
+	# If the flag is --oracle-tns-alias then find the exported variable TNS_ADMIN and map it to the container.  --oracle-tns-alias flag will be passed as it is.
+	if [[ ${argv[${i}]} == "--oracle-tns-alias" ]]
+	then
+		tns_admin=$(env | grep -E '^TNS_ADMIN')	
+		if [[ -n "$tns_admin" ]]; then
+			tns_admin=$(echo $tns_admin | cut -d '=' -f2)
+		else 
+			tns_admin="${ORACLE_HOME}/network/admin"
+		fi
```

Comment:
How about the following approach:

`tns_admin=${TNS_ADMIN:"${ORACLE_HOME}/network/admin"}`